requirements.txt

```diff
@@ -1,4 +1,4 @@
-matplotlib~=3.7.1
-pandas~=2.0.0
-openeo>=0.15.0
-geojson~=3.0.1
\ No newline at end of file
+geojson==2.5.0
+numpy==1.19.5
+openeo==0.22.0
```

Comment:
why is this pinned to version 0.22.0?

latest version is 0.28.0 FYI

Reply:
I think I noted down the current version I am using. Will need to update my openeo and check if still runs